sources/europe/fr/FR-IGN-Ortho.geojson

```diff
@@ -0,0 +1,336 @@
+{
+    "type": "Feature",
+    "properties": {
+        "attribution": {
+            "url": "http://openstreetmap.fr/bdortho",
+            "text": "BDOrtho IGN",
+            "required": true
+        },
+        "description": "French IGN aeral imagery",
+        "url": "http://proxy-ign.openstreetmap.fr/bdortho/{zoom}/{x}/{y}.jpg",
```

Comment:
This URL only gives me HTTP/403 errors, for example when trying to access http://proxy-ign.openstreetmap.fr/bdortho/12/2103/1461.jpg – is that just a temporary hickup or has the URL been changed?